src/OnePlusBot/Modules/Shutdown.cs

```diff
@@ -11,8 +11,7 @@ public class ShutdownModule : ModuleBase<SocketCommandContext>
         [RequireOwner]
         public async Task ShutdownAsync()
         {
-            var emoteTrue = Emote.Parse("<:success:499567039451758603>");
-            await Context.Message.AddReactionAsync(emoteTrue);
+            await Context.Message.AddReactionAsync(Emote.Parse("<:success:499567039451758603>"));
```

Comment:
This does not improve readability and probably does not improve performance either. Revert.